tests/test_api.py

```diff
@@ -2943,3 +2943,32 @@ def labels(self):
             kwargs['flatpak'] = True
             response = osbs._do_create_prod_build(**kwargs)
             assert isinstance(response, BuildResponse)
+
+    def test_do_create_prod_build_invalid_yaml(self, osbs, tmpdir, monkeypatch):  # noqa
+        yaml_file = tmpdir.join('invalid.yaml')
+        yaml_file.write('\n'.join(['hallo: 1', 'bye']))
+
+        def get_repo_info(*_, **__):
```

Comment:
Is this a standardized way of doing `*args, **kwargs` for unused arguments?

Reply:
I don't know, probably not.

Reply:
I'd stay with standard `*args, **kwargs` which are commonly used

Reply:
ok